static/docs/commands-reference/remote_add.md

```diff
@@ -249,6 +249,46 @@ container doesn't already exist, it will be created automatically.
 
 </details>
 
+<details>
+
+### Click for Aliyun OSS
+
+First you need to setup OSS storage on Aliyun Cloud and then use s3 style url for oss storage and make endpoint a configurable value, an example is shown below:
+```
+$ dvc remote add myremote oss://my-bucket/path
+```
+To set key id, key secret and endpoint you need to use modify command from dvc, a sample ussge is show below:
+```
+$ dvc remote modify myremote oss_key_id my-key-id
```

Comment:
_important_: we need to use `--local`. It's a **very bad practice** to put keys into a config file that goes into Git. Is there something similar to `~/.aws/credentials` for OSS - similar to S3?

Reply:
@nanaya-tachibana Can you please throw some light on this?

Reply:
@shcheklein I have added ``--local`` flag in all the code blocks